Updated moonlight-common-c and Various Improvements

dos-ise opened the pull request:
<h2>Updated moonlight-common-c and Various Improvements</h2>

<h3>Description:</h3>
<p>
This pull request introduces several updates and enhancements aimed at improving functionality, security, and compatibility within the project. Below are the key changes:
</p>
<ul>
  <li>Improved GitHub Action workflow by removing warnings for obsolete methods.</li>
  <li>Added a CA bundle for CURL using Emscripten's virtual file system API.</li>
  <li>Fixed validation of non-pinned trusted TLS certificates.</li>
  <li>Installed Java for Emscripten to support Closure Compiler in the pipeline.</li>
  <li>Updated the project to Tizen Studio 6.0 for better compatibility.</li>
</ul>

<h3>Tags:</h3>
<p>@moonlight-stream</p>

dos-ise committed:
Updated moonlight-common-c

dos-ise committed:
Updated to Tizen Studio 6.0

dos-ise committed:
Fixed typo regarding tizen studio 6

dos-ise committed:
Install Java for Emscripten (Closure Compiler support)

dos-ise committed:
Fix validation of non-pinned trusted TLS certificates

https://github.com/moonlight-stream/moonlight-chrome/commit/be7b87797bb0b87d6202f19ef497e9a1cefeb4f9

dos-ise committed:
Use full encryption support

https://github.com/moonlight-stream/moonlight-chrome/commit/2e3643235f7e75a1788f6f0fc924b7d18dbf03c4#diff-ecb13e9404534688be1c88a4c0e33e98b40164a57867bcf770fb0cc82998534e

dos-ise committed:
Add a CA bundle for CURL

https://github.com/dos-ise/moonlight-chrome-tizen_2/commit/232e3c57f3278fab9ff6b0efc2d9d2fbaef3248c#diff-041b28966a7c0cc5f20147b44d9adac3751b1aecb9e7e6ba852a3ca14034c061

dos-ise committed:
Revert sys.mount

dos-ise committed:
Add a CA bundle for CURL to CMakeLists.txt

https://github.com/dos-ise/moonlight-chrome-tizen_2/commit/232e3c57f3278fab9ff6b0efc2d9d2fbaef3248c#diff-041b28966a7c0cc5f20147b44d9adac3751b1aecb9e7e6ba852a3ca14034c061

dos-ise committed:
Add a CA bundle for CURL via  Emscripten's virtual file system API

https://github.com/dos-ise/moonlight-chrome-tizen_2/commit/232e3c57f3278fab9ff6b0efc2d9d2fbaef3248c#diff-041b28966a7c0cc5f20147b44d9adac3751b1aecb9e7e6ba852a3ca14034c061

dos-ise committed:
Fix MAIN_THREAD_EM_ASM FS.Mount

dos-ise committed:
Added missing #include <emscripten.h>

Added missing #include <emscripten.h>

dos-ise committed:
added FORCE_FILESYSTEM to compile options

added FORCE_FILESYSTEM to compile options

dos-ise committed:
Revert FORCE_FILESYSTEM compile options

dos-ise committed:
Update create-release to remove warnings

dos-ise committed:
Use ncipollo/release-action@v1.16.0 for create github release

dos-ise committed:
Added checkout repo for readme.md

dos-ise committed:
Removed redunant upload-release-asset

dos-ise committed:
Added release_body.md

dos-ise committed:
Use correct md file for Release

OneLiberty commented:
Quite a lot of changes here. 
Can you tell me what test you've done. 

From looking at it, i saw that you removed some code from the lib specific to our plateform, so i have my doubt on this working as before. 

I'll try to take a bit of time to test it by the end of the week.

dos-ise commented:
Thanks, can point me to the removed parts?

Maybe i missed something while merging common. 

I am running this since the creation of this request on my tizen 6.5 TV.

OneLiberty commented:
Sorry forgot to answer you. 
I was thinking about the __EMSCRIPTEN__ compile tag in the library

AlexPresso commented:
Hi @dos-ise 
These changes look very nice, I'll test it, I was just wondering, does your port includes 120 Hz / 144 Hz support ? :)

zenaror committed:
Ajustes HDR

zenaror committed:
Update config.xml

zenaror committed:
ajustes

dos-ise committed:
Updated to tizen-studio_6.1

dos-ise committed:
Trigger

dos-ise committed:
Revert to 6.0

dos-ise committed:
Upgrade Tizen Studio to version 6.1 and adjust setup

# Extract the WGT manually (wgt = zip)
RUN mkdir -p /home/moonlight/userwidget \
    && unzip /home/moonlight/Moonlight.wgt -d /home/moonlight/userwidget

# Package the extracted widget into a USB installable ZIP
RUN cd /home/moonlight && zip -r MoonlightUSB.zip userwidget

dos-ise committed:
Simplfied cmake for opus

dos-ise committed:
Force opus into embedded mode

dos-ise committed:
Remove vendored Opus source

dos-ise committed:
Add Opus 1.5.2 as submodule

dos-ise committed:
Added Init submodules

dos-ise committed:
Updated h264bitstream

dos-ise committed:
Improved  run-and-upload:

dos-ise committed:
set up image tag correct for docker publish

dos-ise committed:
Revert Common for testing

dos-ise committed:
Merge branch 'samsung_wasm' of https://github.com/dos-ise/moonlight-chrome-tizen_2 into samsung_wasm

dos-ise committed:
Revert http

dos-ise committed:
Updated h264 lib again
